packages/documentation/src/stories/internet-header/getting-started.docs.mdx

```diff
@@ -106,7 +80,9 @@ If you're working on a new project, you probably need a new configuration. Pleas
   href="mailto:digitalconsulting@post.ch"
   target="_blank"
   rel="noopener"
->Contact Post Portal Team <post-icon name="1001" className="fs-large" /></a>
+>
+  Contact Post Portal Team <post-icon name="1001" className="fs-large" />
+</a>
```

Comment:
This format will lead to a `<p>` tag around the inner text and the `<post-icon>`.